website/addons/base/exceptions.py

```diff
@@ -22,3 +30,38 @@ def renderable_error(self):
         please report it to <a href="mailto:support@osf.io">support@osf.io
         </div>
         '''
+
+    @property
+    def render_error(self):
+        # TODO Refactor me to be all in the front end
+        # 2/10/14 ping @chrisseto when refactoring
+        additional = ''
+        if not self.can_download:
+            additional += "<style>.fileDownload{display: none;}</style>"
+
+        if not self.can_delete:
+            additional += "<style>.fileDelete{display: none;}</style>"
+
+        return self.renderable_error + additional
+
+
+class FileDeletedError(AddonEnrichmentError):
+
+    @property
+    def renderable_error(self):
```

Comment:
Oh, I see--`renderable_error` is a hook for subclasses to override. Perhaps this would be more clear if we rename `renderable_error -> _renderable_error`. A docstring would help as well.